bugbug/repository.py

```diff
@@ -1000,7 +1000,7 @@ def clone(repo_dir):
     if proc.returncode:
         raise hglib.error.CommandError(cmd, proc.returncode, out, err)
 
-    logger.info("mozilla-central cloned")
+    logger.info(f"{url} cloned")
```

Comment:
Nit for consistency:
```suggestion
    logger.info(f"{repo_dir} cloned")
```